ui/src/router/index.js

```diff
@@ -209,12 +210,34 @@ router.beforeEach((to, from, next) => {
 
   if (hasLoggedIn) {
     if (to.name == "general-logout") {
-      store.dispatch("user/setUserData", {});
-      store.dispatch("user/setToken", "");
-      store.dispatch("user/setItems", []);
-      Cookies.remove("access_token_cookie");
+      // note: then and catch blocks below are needed
```

Comment:
I think you dont have to keep these comments here, unless its needed for future development XD